apps/admin/backend/jest.config.js

```diff
@@ -12,19 +12,21 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/test/setupTests.ts'],
   coverageThreshold: {
     global: {
-      statements: 98,
-      branches: 91,
-      functions: 100,
-      lines: 98,
+      statements: 97,
+      branches: 89,
+      functions: 99,
+      lines: 97,
     },
   },
+  coverageProvider: 'v8',
```

Comment:
All of the branching logic added in the `store` makes source map computation too complicated (see [thread](https://votingworks.slack.com/archives/C014MGC0WRE/p1685128457266709)) and can crash testing when run with `--coverage`. `v8` is an alternate coverage provider introduced in Jest 26 that has some limitations (cannot handle istanbul ignore so, for us, may not get to full coverage) but is way faster.